utils/arp-whisper/files/etc/arp-whisper.ini

```diff
@@ -0,0 +1,8 @@
+logging_level = "info"
```

Comment:
Having .ini as extension is quite unsual in OpenWrt.

Reply:
This is a configuration file, so I'm okay with it. But IMO it would be better to put it into a subdirectory like `/etc/arp-whisper/config.ini`